mc2/controllers/docker/models.py

```diff
@@ -28,10 +29,11 @@ def get_marathon_app_data(self):
         parameters_dict = []
         if self.volume_needed:
             parameters_dict.append({"key": "volume-driver", "value": "xylem"})
+            generic_volume_name = "%s_media" % self.app_id
             parameters_dict.append({
                 "key": "volume",
-                "value": "%(app_id)s_media:%(path)s" % {
-                    'app_id': self.app_id,
+                "value": "%(volume_name)s:%(path)s" % {
```

Comment:
This might do for now but we should maybe have some way to check uniqueness and prevent collisions in a multi-tenant environment